.github/workflows/docker-build-and-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: use node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '18.x'
+          node-version: "18.x"
```

Comment:
Will this get undone by another developer checkin? what settings do you have enabled?

Reply:
No I believe it should be double quotes for yaml files.